src/validatedODEs.jl

```diff
@@ -2,6 +2,33 @@
 
 const _DEF_MINABSTOL = 1.0e-50
 
+
+"""
+    TMSol{N,T}
+
+Structure containing the solution of a validated integration.
+
+# Fields
+`time :: Vector{T}`  Vector containing the expansion time of the `TaylorModel1` solutions
+
+`fp   :: Vector{IntervalBox{N,T}}`  IntervalBox vector representing the flowpipe
+
+`xTMv :: Matrix{TaylorModel1{TaylorN{T},T}}`  Matrix whose entry `xTMv[i,t]` represents 
+    the `TaylorModel1` of the i-th dependent variable, obtained at time time[t].
+"""
+struct TMSol{N,T<:Real}
```

Comment:
here we can use `TMSol{N, VN<:AbstractVector{N}, T<:Real}` so use `time::VN` etc